.buildkite/scripts/steps/code_coverage/ftr_configs.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+.buildkite/scripts/bootstrap.sh
+.buildkite/scripts/build_kibana_plugins.sh
```

Comment:
@spalger Is there a way to not build the plugins everytime?  @dmlemeshko and I think we should fix this in a follow on pr.

Reply:
Oh ok, then I guess the next question is how do we "connect" them?
@dmlemeshko this is what you were wondering about correct?

Reply:
See how we do it in `base.yml`, we run the build in one step, then at the end of that step we use `buildkite-agent artifact upload` ([docs](https://buildkite.com/docs/agent/v3/cli-artifact#uploading-artifacts)) and then download that artifact by name in other steps with `buildkite-agent artifact download`. Don't forget to add a `depends_on` for that step so that your workers won't start up until the artifact is uploaded.

Reply:
Ahhh, I see said the blind man. 
Thanks @spalger 
Whatcha think @dmlemeshko

Reply:
@dmlemeshko and I were just discussing this.  @spalger have you tried doing this with source, or only built kibana?

Reply:
I don't understand the question.